### ISSUE TITLE
Separation of unknown speakers

### ISSUE THREAD

**Issue by srdfjy** [speechbrain/speechbrain#982]
hi，Does sepformer support the separation of unknown speakers? (That is, the speakers in the test set are not in the training set). I haven't seen the relevant introduction. If not, which models of speechbrain support the separation of unknown speakers, thank you!

**Comment by mravanelli** [comment]
Sure,
all the available models for separation (e.g, sepformer, convtasnet, etc)
support inference on speakers which are not in the test set.

**Comment by srdfjy** [comment]
hi @mravanelli ，I run the default recipe, the speaker in the test set is not in the training set, and sisnr only has 2 points. Is there any special configuration required? Thanks!

**Comment by mravanelli** [comment]
Here Cem could be more precise. But the official performance metrics that
people report in the papers are all derived from test sets with different
speakers. These days, we are working on speech separation on real data and
it seems that the sepformer works well in this setting as well.

**Comment by ycemsubakan** [comment]
The current recipe does not support variable number of speakers, but this is a feature we will add in the near future.

**Comment by srdfjy** [comment]
hi，If I use "create-speaker-mixtures (https://www.merl.com/demos/deep-clustering/create-speaker-mixtures.zip)" to generate a data set, do I need to use "sepformer-customdataset.yaml" Is this configuration to run the recipe?

**Comment by ycemsubakan** [comment]
No, you can just use the WSJ0Mix recipe in this case. You can refer to the README.md in the WSJ0Mix recipe to see what command to run.

**Comment by srdfjy** [comment]
hi, what is the usage scenario of "https://github.com/speechbrain/speechbrain/tree/develop/recipes/WSJ0Mix/separation/hparams/sepformer-customdataset.yaml"?

**Comment by ycemsubakan** [comment]
Hello, this is if you have a dataset other than the supported recipes we currently have (WSJ0mix, WHAM!, WHAMR!, LibriMix)
